e2e/deployers/deployer.go

```diff
@@ -0,0 +1,16 @@
+package deployers
+
+import "github.com/ramendr/ramen/e2e/workloads"
+
+// Deployer interface has methods to deploy a workload to a cluster
+type Deployer interface {
+	Init()
+	Deploy(workloads.Workload) error
+	Undeploy(workloads.Workload) error
+	// Scale(Workload) for adding/removing PVCs; in Deployer even though scaling is a Workload interface
+	// as we can Kustomize the Workload and change the deployer to perform the right action
+	// Resize(Workload) for changing PVC(s) size
+	// Health(Workload) error
+
+	GetID() string
```

Comment:
(nit) Should this just be renamed to GetName instead? ID seems to suggest an identifier, and current uses are to get a name. (again can be done later as we expand its usage across)

Reply:
you are correct. In my previous implementation, GetName was already used for another purpose, so I choose GetID for this. And later on GetName func was removed, so now I could change GetID to GetName.

Updated to GetName.